src/WebJobs.Script/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -60,6 +61,40 @@ public static bool IsLegacyBlobTriggerFunction(this FunctionMetadata metadata)
             return false;
         }
 
+        /// <summary>
+        /// Gets the function group for the specified <see cref="FunctionMetadata"/>.
+        /// </summary>
+        /// <param name="metadata">The function metadata.</param>
+        /// <returns>The function group for this metadata or <c>null</c> if no group specified.</returns>
+        public static string GetFunctionGroup(this FunctionMetadata metadata)
+        {
+            if (metadata == null)
+            {
+                throw new ArgumentNullException(nameof(metadata));
+            }
+
+            if (metadata.Properties.TryGetValue(FunctionGroupKey, out object group))
+            {
+                return group.ToString();
+            }
+
+            // Ensure http and durable triggers are grouped together.
+            foreach (BindingMetadata binding in metadata.InputBindings)
+            {
+                if (binding.IsHttpTrigger())
```

Comment:
Should we also check `binding.Direction == BindingDirection.In` ?

Reply:
I didn't add that check as the original `IsHttpTriggerFunction` extension does not check for that. I figured it was safe to assume a trigger binding is implicitly `in` direction. Is there any way it can be `out`? Do we think it was a bug to not check for this originally?

Reply:
Going to skip adding this check, as `BindingMetadata.IsTrigger` check itself does not verify binding direction. We can consider adding this later as we would have to evaluate the impact on it broader than this feature.